Modules/_zstd/_zstdmodule.h

```diff
@@ -0,0 +1,214 @@
+#pragma once
+/*
+Low level interface to Meta's zstd library for use in the `zstd` Python library.
```

Comment:
```suggestion
Low level interface to Meta's zstd library for use in the zstd Python library.
```
No need for this in comments